share/spice/news/news.js

```diff
@@ -93,10 +93,27 @@
                 meta: {
                     count: goodStories.length,
                     searchTerm: searchTerm,
-                    itemType: 'News articles'
+                    itemType: 'News articles',
+                    rerender: [
+                        'image'
+                    ]
                 },
                 templates: {
                     item: 'news_item'
```

Comment:
Should we switch to the new-er `Media` template? We use it for [Dogo News](https://duckduckgo.com/?q=dogo+news&t=ffab&ia=kidsnews).